### PR TITLE
feat(vue): add dedicated Vue package

### PR DESCRIPTION
**Pull request by waterplea**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes # 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: projects/vue/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# @maskito/vue
+
+[![npm version](https://img.shields.io/npm/v/@maskito/react.svg)](https://npmjs.com/package/@maskito/react)
+[![npm bundle size](https://img.shields.io/bundlephobia/minzip/@maskito/react)](https://bundlephobia.com/result?p=@maskito/react)
```
Comment (nsbarsukov) [inline projects/vue/README.md:4]:
```suggestion
[![npm bundle size](https://img.shields.io/bundlephobia/minzip/@maskito/vue)](https://bundlephobia.com/result?p=@maskito/vue)
```

File: projects/vue/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# @maskito/vue
+
+[![npm version](https://img.shields.io/npm/v/@maskito/vue.svg)](https://npmjs.com/package/@maskito/react)
```
Comment (nsbarsukov) [inline projects/vue/README.md:3]:
```suggestion
[![npm version](https://img.shields.io/npm/v/@maskito/vue.svg)](https://npmjs.com/package/@maskito/vue)
```

File: projects/vue/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+[![npm bundle size](https://img.shields.io/bundlephobia/minzip/@maskito/vue)](https://bundlephobia.com/result?p=@maskito/react)
```
Comment (nsbarsukov) [inline projects/vue/README.md:4]:
```suggestion
[![npm bundle size](https://img.shields.io/bundlephobia/minzip/@maskito/vue)](https://bundlephobia.com/result?p=@maskito/vue)
```

**Review by nsbarsukov — APPROVED** [review]

**Review by sean-perkins — APPROVED** [review]
The proposed Vue directive implementation looks accurate/correct to me 👍

**Pull request merged**